toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -361,3 +367,27 @@ func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safec
 
 	return nil
 }
+
+func addCustomizerRelease(imageChroot *safechroot.Chroot) error {
+	var err error
+
+	logger.Log.Infof("Creating image customizer release file")
+
+	// TODO: update micVersion once the mic release process is in place
+	micVersion := "0.0.0"
+	currentTime := time.Now().Format("2006-01-02T15:04:05Z")
+
+	customizerReleaseFilePath := filepath.Join(imageChroot.RootDir(), "/etc/mariner-customizer-release")
+	lines := []string{
+		fmt.Sprintf("%s=\"%s\"", "VERSION", micVersion),
```

Comment:
Probably should call this `TOOL_VERSION` to avoid it being confused for an image version.